google/resumable_media/_download.py

```diff
@@ -466,3 +472,28 @@ def get_range_info(response, get_headers, callback=_helpers.do_nothing):
         int(match.group(u'end_byte')),
         int(match.group(u'total_bytes'))
     )
+
+
+def check_for_zero_content_range(response, get_status_code, get_headers):
```

Comment:
This method should probably be made private. Also, it it is used only once, then encapsulation is not necessary, and the check can be placed inside the `_process_response()` method.

Reply:
Yeah, making this function as a private function make sense. 
I agree, it is currently used once but we may need it somewhere in future. Also, making it as a separate function, will make _process_response() method more cleaner and shorter. Also, it helps to write clean and separate tests for it.

Reply:
I don't disagree with this but let's see what Googlers have to say.